### DISCUSSION TITLE
Limiting users to everything except code in organization

### DISCUSSION THREAD

**Discussion by Interghost** [community/community#45355] [New to GitHub]
Hi,

Is it possible to limit users to be able to access everything (issues/projects/discussions/wikis) except code/pull requests in an organization's private repo? 
Basically the idea is to add non-tech and/or support personal to the above but without giving them any access to code (even read-only).

**Comment by jge162** [comment]
Yes, a feature called "Repository Permissions" that allows you to set access levels for different types of users and teams within an organization.

1. Create a new team in your organization on GitHub.
2. In the repository settings, go to the "Collaborators & teams" tab.
3. Add the team you created in step 1 with read-only access to the repository.
4. Add the non-technical and/or support personnel to the team created.
5. 👍

**Reply by Interghost** [reply]
> Yes, a feature called "Repository Permissions" that allows you to set access levels for different types of users and teams within an organization.
> 
>     1. Create a new team in your organization on GitHub.
> 
>     2. In the repository settings, go to the "Collaborators & teams" tab.
> 
>     3. Add the team you created in step 1 with read-only access to the repository.
> 
>     4. Add the non-technical and/or support personnel to the team created.
> 
>     5. 👍

but this still gives them access, its just read-only.. I don't want them to have code access

**Reply by jge162** [reply]
@Interghost i think harrander is a bot. They just copy pasted what I sent to you. FYI. Account just created 1 hour ago. 


But, Use GitHub's "Limited Access" feature: This feature allows you to invite someone to the repository and give them access to specific areas, such as issues, pull requests, and project boards, but not the code.

update** feature removed.

**Reply by Interghost** [reply]
@jge162 where do I find this feature

**Reply by jge162** [reply]
FYI though. this feature is available for private repositories only, and it's not available for public repositories.

**Reply by Interghost** [reply]
hm.. I don't have this, when I type in the user I'm presented with the option choose role:
read/triage/write/maintain/admin

**Reply by jge162** [reply]
Feature is gone, I just tried myself. Doesn’t appear to be an option to not see code for a maintainer or service member you wanted to add. Sorry 😞 it was deprecated. Read only is lowest access.

**Reply by Interghost** [reply]
I don't get it why would this be deprecated :| It basically means you must use a secondary tool for project management... which in turn means why to use Github for it at all, just use the 2nd tool alone.

Anyway, thank you for trying to help.

**Reply by jge162** [reply]
I know it’s not the answer you wanted. But, nice meeting you. Best to you 👍
